AsyncDisplayKit/ASCellNode.h

```diff
@@ -30,6 +30,14 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
    * Indicates a cell is no longer visible
    */
   ASCellNodeVisibilityEventInvisible,
+  /**
+   * Indicates user has started dragging the visible cell
+   */
+  ASCellNodeVisibilityEventWillBeginDragging,
```

Comment:
Feedback needed:

Adding 2 events here to listen to dragging events for visible cells on screen.  I am not sure if they belong in visible events in the first place, but it seems this is the easiest thing to do to make it happen, because they function similarly to scrollviewDidScroll and update the visible cell rect for visible cells (ASCellNodeVisibilityEventVisibleRectChanged).  

Alternatively we can create a separate `UserInteractionEvent` for drag events, but that would create a lot more code and mess.  Would like to hear your feedback on this.